### DISCUSSION TITLE
Why do I set 'LongPathsSupported=1' in the system to be useless for pnpm

### DISCUSSION THREAD

**Discussion by WangJincheng4869** [pnpm/pnpm#7718] [General]
Windows 10 already supports long path settings, but it seems to have no effect on PNPM.
[Maximum Path Length Limitation](https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=powershell)

**Comment by RexSkz** [comment]
It may be related to this issue in Node.js: https://github.com/nodejs/node/issues/50753. There is a PR but it's not merged yet: https://github.com/nodejs/node/pull/51097

**Reply by WangJincheng4869** [reply]
Okay, thank you! It seems that this issue has been on hold for a long time, and we look forward to their update.
